.github/workflows/ami-release.yml

```
@@ -17,3 +17,17 @@ jobs:
       - name: Build AMI
         run: |
           packer build -var-file="development-arm.vars.pkr.hcl" -var-file="common.vars.pkr.hcl" amazon-arm64.pkr.hcl
+
+      - name: Merging migration files
+        run: cat $(ls -1) > ../migration-output.sql
+        working-directory: ${{ github.workspace }}/migrations/db/migrations
+
+      - name: Push migration files to S3
+        uses: jakejarvis/s3-sync-action@master
```

```suggestion
        uses: jakejarvis/s3-sync-action@master
        with:
          args: --delete
```

Might want to replace existing files

---

ok, added.